mmdet/integration/nncf/compression.py

```diff
@@ -155,6 +155,8 @@ def dummy_forward(model):
                                                       resuming_state_dict=resuming_state_dict)
     model = change_export_func_first_conv(model)
 
+    model.export = getattr(model, 'export')
```

Comment:
Please, note that it will require addition `from functools import partial` at the begin of the file
```suggestion
    model.export = partial(model.export, network = model)
```